Include jackson-dataformat-xml dependency.

From: Lars Francke <git@lars-francke.de>

This allows us to use XmlLayout for Log4jV2.
By including it here as a dependency we can make sure that we always have
the matching version and we don't need to include it manually later in the
build.
---
 0 files changed

diff --git a/server/pom.xml b/server/pom.xml
index 410b51480e..b7dcf46111 100644
--- a/server/pom.xml
+++ b/server/pom.xml
@@ -205,6 +205,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+          <!-- This is an optional dependency of log4j which is needed to use XmlLayout -->
+          <groupId>com.fasterxml.jackson.dataformat</groupId>
+          <artifactId>jackson-dataformat-xml</artifactId>
+        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-joda</artifactId>
